src/components/CreateCommentContainer.js

```diff
@@ -0,0 +1,51 @@
+import React, { PureComponent } from 'react';
+import CommentsForm from './CommentsForm';
+import Comments from './Comments';
+import { connect } from 'react-redux';
+import { createComment } from '../actions/commentActions';
+import PropTypes from 'prop-types';
+
+const mapDispatchToProps = dispatch => ({
+  onSubmit(postId, comment) {
```

Comment:
definitely try to separate your redux code and your display components into separate files, as connect will associate the two and it leads to more readable code